scout/parse/variant/variant.py

```diff
@@ -418,6 +424,93 @@ def set_str_info(variant: Variant, parsed_variant: Dict[str, Any]):
     parsed_variant["str_inheritance_mode"] = call_safe(str, variant.INFO.get("InheritanceMode"))
 
 
+def set_fusion_info(variant: Variant, parsed_variant: Dict[str, Any]):
+    """Add Fusion information if present."""
+
+    def replace_nan(value: str, nan_value: str = "nan", replace_by: Any = "") -> str:
+        if value == nan_value:
+            return replace_by
+        else:
+            return value
+
+    parsed_variant["gene_a"] = call_safe(str, variant.INFO.get("GENEA", ""))
+    parsed_variant["gene_b"] = call_safe(str, variant.INFO.get("GENEB", ""))
+    parsed_variant["tool_hits"] = call_safe(str, variant.INFO.get("TOOL_HITS", 0))
+    parsed_variant["fusion_score"] = call_safe(str, variant.INFO.get("SCORE", None))
+    parsed_variant["hgnc_id_a"] = call_safe(int, variant.INFO.get("HGNC_ID_A", 0))
+    parsed_variant["hgnc_id_b"] = call_safe(int, variant.INFO.get("HGNC_ID_B", 0))
+    parsed_variant["orientation"] = call_safe(str, variant.INFO.get("ORIENTATION", ""))
+    parsed_variant["frame_status"] = call_safe(
+        str, replace_nan(variant.INFO.get("FRAME_STATUS", ""))
+    )
+    parsed_variant["transcript_id_a"] = call_safe(
+        str, replace_nan(variant.INFO.get("TRANSCRIPT_ID_A", ""))
+    )
+    parsed_variant["transcript_id_b"] = call_safe(
+        str, replace_nan(variant.INFO.get("TRANSCRIPT_ID_B", ""))
+    )
+    parsed_variant["exon_number_a"] = call_safe(
+        str, call_safe(int, replace_nan(variant.INFO.get("EXON_NUMBER_A", "")))
+    )
+    parsed_variant["exon_number_b"] = call_safe(
+        str, call_safe(int, replace_nan(variant.INFO.get("EXON_NUMBER_B", "")))
+    )
+    parsed_variant["chr_a"] = call_safe(str, variant.INFO.get("CHRA", ""))
+    parsed_variant["chr_b"] = call_safe(str, variant.INFO.get("CHRB", ""))
+    parsed_variant["position_a"] = call_safe(str, variant.INFO.get("POSA", ""))
+    parsed_variant["position_b"] = call_safe(str, variant.INFO.get("POSB", ""))
+    parsed_variant["breakpoint_a"] = f"{parsed_variant['chr_a']}:{parsed_variant['position_a']}"
```

Comment:
Check the last variant in the demo variants. It has not much info about the coordinates, and breakpoints 0:0 doesn't make much sense. Perhaps replace 0 with "-" ? 

<img width="130" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/de0d87d8-e32a-406d-99b9-675ed0fc8ccf">

Also consider adding "-" instead of nothing for the missing positions of variant 2 and 3

Reply:
I've realised I can get the break points from the coordinates position and end so I we don't need to store the breakpoint information. I will update this and remove it

Reply:
Removed breakpoint parsing in favour of postions, now all fusions should display breakpoints.